Classes/Private/Manager/Database/OCTRealmManager.h

```diff
@@ -51,6 +53,7 @@
 
 - (OCTFriend *)friendWithFriendNumber:(OCTToxFriendNumber)friendNumber;
 - (OCTChat *)getOrCreateChatWithFriend:(OCTFriend *)friend;
+- (OCTCall *)getOrCreateCallWithFriendNumber:(OCTToxFriendNumber)friendNumber;
```

Comment:
Even more convenient it would be to getOrCreate call with chat. In case of group calls we wouldn't even have to change API.